src/styles/rating.css

```
@@ -0,0 +1,20 @@
+.rating {
+  display: flex;
+}
+
+.rating__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.rating--5 .rating__star,
+.rating--1 .rating__star:first-child,
```

В моїй роботі також був first-child, але ментор запропунував зробити так:
```suggestion
.rating--1 .rating__star:nth-child(1),
```

---

That's not crucial, I actually prefer `:first-child` here